### ISSUE TITLE
Virtual property in abstract class with no setter should raise a compiler error

### ISSUE THREAD

**Issue by TobiasSchulte** [dotnet/roslyn#7025]
The following program produces different results in the VS2015 compiler and VS2013 compiler:

``` csharp
using System;

namespace ConsoleApplication1
{
    abstract class MyClass
    {
        public virtual int MyProperty { get; }
    }

    public class Program
    {
        static void Main(string[] args)
        {

            Console.ReadLine();
        }
    }
}
```

The 2015 compiler compiles with no errors or warnings:
```
D:\Projects\ConsoleApplication1>"C:\Program Files (x86)\MSBuild\14.0\Bin\csc.exe" Program.cs
Microsoft (R) Visual C# Compiler version 1.0.0.50618
Copyright (C) Microsoft Corporation. All rights reserved.
```

The 2013 compiler produces the following output:
```
D:\Projects\ConsoleApplication1>"C:\Windows\Microsoft.NET\Framework\v4.0.30319\csc.exe" Program.cs
Microsoft (R) Visual C# Compiler version 4.6.0081.0
for C# 5
Copyright (C) Microsoft Corporation. All rights reserved.

This compiler is provided as part of the Microsoft (R) .NET Framework, but only supports language versions up to C# 5, which is no longer the latest version. For compilers that support newer versions of the C# programming language, see http://go.microsoft.com/fwlink/?LinkID=533240

Program.cs(7,43): error CS0840: 'ConsoleApplication1.MyClass.MyProperty.get' must declare a body because it is not marked abstract or extern. Automatically implemented properties must define both get and set accessors.
```

**Comment by leppie** [comment]
Nice catch :+1:

**Comment by leppie** [comment]
I dont think that is a valid error, at least not anymore. 
```
class Foo : MyClass {
    public override int MyProperty => 1;
}
```

**Comment by jskeet** [comment]
Agreed. This is simply a feature of C# 6 making previously-invalid code valid. You could find a huge number of examples where that's the case.

**Comment by comdiv** [comment]
Agreed. New properties are more comfortable to use. Initializers and method-bodies  are valid shortcuts.
